web/src/client/index_test.ts

```diff
@@ -268,6 +321,29 @@ spec:
   contour:
     version: ""
 `);
+  });
+
+  describe("/installer/latest?resolve=true", () => {
+    before(async () => {
+      await client.postInstaller(latest);
+    });
+
+    it("returns yaml with version", async () => {
+      const yaml = await client.getInstallerYAML("latest", true);
```

Comment:
doesn't look like that second parameter is used in `getInstallerYAML`

Reply:
I forgot to implement that in the client. Added now.